src/Components/People.js

```diff
@@ -0,0 +1,73 @@
+import axios from "axios";
+import React from "react";
+
+class People extends React.Component {
+  state = { input: "", people: [] };
+
+  fetchPeople = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/people/");
+      debugger;
+      this.setState({ people: res.data });
+      debugger;
+    } catch (error) {
+      this.setState({ people: [] });
+    }
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    this.fetchPeople();
+    this.setState({ input: "" });
+  };
+
+  handleChange = (e) => {
+    this.setState({ input: e.target.value });
+  };
+
+  render() {
+    const { input, people } = this.state;
+
+    const getPersonsDeets = (people) => {
```

Comment:
This function makes it so you show all of the people at once. This makes it so the test case passes, but this isn't really what we are looking for.